graph/src/blockchain/client.rs

```diff
@@ -11,13 +11,17 @@ use anyhow::anyhow;
 // Substreams only requires the FirehoseEndpoints.
 #[derive(Debug)]
 pub enum ChainClient<C: Blockchain> {
-    Firehose(FirehoseEndpoints),
+    Firehose(FirehoseEndpoints, Option<C::Client>),
```

Comment:
I think we had this code before and was removed, did you intentionally add the client back in?

Reply:
Will remove it

Reply:
Removed